fake-snippets-api/lib/db/schema.ts

```diff
@@ -100,6 +100,39 @@ export const orderFileSchema = z.object({
 })
 export type OrderFile = z.infer<typeof orderFileSchema>
 
+export const orderQuoteComponentSchema = z.object({
+  component_id: z.string(),
+  name: z.string(),
+  manufacturer_part_number: z.string().nullable(),
+  jlcpcb_part_number: z.string().nullable(),
+  quantity: z.number(),
+  unit_price: z.number(),
+  total_price: z.number(),
+  type: z.string(), // e.g., "resistor", "capacitor", etc.
+  available: z.boolean().default(true),
```

Comment:
Instead of jlcpcb_part_number, can you use supplier_part_numbers

no need for the type property